plan/logical_plan_test.go

```diff
@@ -158,6 +158,27 @@ func mockResolve(node ast.Node) (infoschema.InfoSchema, error) {
 			},
 			State: model.StatePublic,
 		},
+		{
+			Name: model.NewCIStr("e_d_c_str"),
```

Comment:
It's hard to read the test code if we can't tell from the index name that `c` is a prefix column